apps/vr-tests/package.json

```diff
@@ -27,6 +27,7 @@
     "@fluentui/react-hooks": "*",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28636&commitId=25147239e9312e25b66a6aa8b1a9a6fceb127017&env=prod&clientType=FLUENTUI)


  <details><summary><b>PieChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PieChart.Root.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>